x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_related_list.tsx

```diff
@@ -214,9 +214,9 @@ export const ProcessEventListNarrowedByType = memo(function ProcessEventListNarr
           eventCategory: `${eventType}`,
           eventType: `${event.ecsEventType(resolverEvent)}`,
           name: event.descriptiveName(resolverEvent),
-          entityId,
+          entityId: String(entityId),

```

Comment:
the behavior here is changed. Before, if the event was a legacy event and the id was `undefined` or `0`this would be `''`. Now it will be `'0'` or `'undefined'`. I'm not sure that either behavior is correct.

Reply:
If it's `undefined` I wonder if we should mark it as an empty string here. I would think `'0'` would be fine if that is what was sent by the sensor.

Reply:
Hmm also the field name here is a bit misleading, we're setting `entityId` with `eventId`. Is that intentional?

Reply:
Went to track down what its used for. its not used. removing it.